recipes/unifrac/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: unifrac
+  version: "0.9.1"
+
+source:
+  git_url: https://github.com/biocore/unifrac.git
+  git_rev: 0.9.1
+
+build:
+  # noarch_python: True
+  preserve_egg_dir: True
+  number: 1
+
+requirements:
+  build:
+    - python 3.5*
+    - gcc  # [not osx]
+    - llvm  # [osx]
+    - hdf5
```

Comment:
This should be: `hdf5 1.8.17`, then anything using this package doesn't need to worry about mismatched ABIs.